src/data/showcases.js

```diff
@@ -1104,6 +1104,15 @@ export const Showcases = [
     source: "https://github.com/cardano-foundation/cardano-governance-voting-tool",
     tags: ["governance", "opensource"],
   },
+  {
+    title: "Dedium",
+    description:
+      "Use the power of your idle GPU for AI and rendering workloads.",
```

Comment:
```suggestion
      "Share the power of your idle GPU for AI and rendering workloads.",
```
Perhaps @0xBora this simple change would resolve the ambiguity?